css/_base.scss

```diff
@@ -108,14 +108,15 @@ form {
 }
 
 .leftwatermark {
-    left: $defaultToolbarSize;
-    margin-left: 10px;
+    left: 32px;
+    top: 32px;
     background-image: url($defaultWatermarkLink);
     background-position: center left;
 }
 
 .rightwatermark {
-    right: 15;
+    right: 32px;
+    top: 32px;
```

Comment:
<img width="244" alt="screenshot 2018-07-20 14 37 31" src="https://user-images.githubusercontent.com/3264676/43021815-a9a2a3f6-8c2a-11e8-990f-f151bb5bdf54.png">

The right watermark appears under the local video. This bug has not been introduced by this pr, but if we're making changes we may as well fix the positioning. Also it doesn't appear on the welcome page at all when enabled. When I look at the code I believe it should appear, but I may be wrong.

I don't see "powered by" label when enabled either.